subsys/net/ip/net_tls_internal.h

```diff
@@ -40,13 +41,18 @@
 
 #endif /* CONFIG_MBEDTLS */
 
+void net_tls_init(void);
```

Comment:
I have tried to document all the functions (both internal and external) in header files using doxygen comments. Lets try to follow this rule in these patches too.